modules/core/test/test_ds.cpp

```diff
@@ -50,7 +50,7 @@ static void cvTsSimpleSeqShiftAndCopy( CvTsSimpleSeq* seq, int from_idx, int to_
 
     if( from_idx == to_idx )
         return;
-    assert( (from_idx > to_idx && !elem) || (from_idx < to_idx && elem) );
+    CV_Assert( (from_idx > to_idx && !elem) || (from_idx < to_idx && elem) );
```

Comment:
Please use this workaround:

```
    if (elem)
        CV_Assert(from_idx < to_idx);
    else
        CV_Assert(from_idx > to_idx);
```

Reply:
@alalek 
Thanks for your review. I fixed them.